pkg/io/fileWriter_test.go

```diff
@@ -0,0 +1,25 @@
+package io
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMakeDirForFile_HappyPath(t *testing.T) {
+	var filePath = "./log/test.log"
```

Comment:
Honestly tried to play with that `ioutil.TempFile`. We use it in boltdb test. So we don't remove this temp folder and file it creates. Since you don't have enough permission. Even code looks more complicated for simple unit test with happy path. Does it make sense to write it this way?